main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1748,6 +1748,13 @@ public void setContent() {
                     }
                     ImagesActivity.startActivity(activity, cache.getGeocode(), cache.getSpoilers());
                 });
+
+                // if there is only a listing background image without other additional pictures, change the text to better explain the content.
+                if (cache.getSpoilers().size() == 1 && getString(R.string.cache_image_background).equals(cache.getSpoilers().get(0).title)) {
```

Comment:
not sure, but will we retrieve the cache page as a localized version, so that we can compare against a localized `R.string.cache_image_background`? Or will it always be an English string we have to look for?

Reply:
we explicitly set this string resource (localized) as title when parsing the background. So as long as the user doesn't change the device setting, this check will work.

Reply:
Ah, thanks, didn't know that - always learning something new while working on the code...